src/app/app.component.spec.ts

```diff
@@ -20,11 +20,11 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it(`should have as title 'demostore'`, () => {
+  /* it(`should have as title 'demostore'`, () => {
     const fixture = TestBed.createComponent(AppComponent);
     const app = fixture.componentInstance;
     expect(app.title).toEqual('demostore');
-  });
+  }); */
```

Comment:
Commenting out tests is a no-go. Even without talking about tests yet this should be easy to adjust:
expect(app.title).toEqual('demostore');